cypress/support/types.ts

```diff
@@ -0,0 +1,11 @@
+export const selectors = {
+  class: {
+    outputText: '.output-text',
+    reactMonacoEditor: '.react-monaco-editor-container'
+  },
+  id: {},
+  data: {},
+  tag: {
+    inputSolution: 'input[name="solution"]'
+  }
+};
```

Comment:
This would make more sense if it was called 'selectors', since it doesn't contain any types.

Reply:
Oops, I meant the entire file. It should be called `selectors.ts`, imo.